src/main/java/Duke.java

```diff
@@ -1,10 +1,69 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static Task[] tasks = new Task[100];
+    static String horizontalLine = "____________________________________________________________\n";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printGreetings();
+        boolean shouldAbort = false;
+
+        while(!shouldAbort) {
+            Scanner in = new Scanner(System.in);
+            String userInput = in.nextLine().trim();
+            String[] taskCommand = userInput.split(" ", 2);
+
+            switch (taskCommand[0]) {
+                case "bye":
```

Comment:
Switch Case : For each case statement no indent is need (Coding Standard)
switch (condition) {
case ABC:
    statements;
    // Fallthrough

Reply:
Good catch! I thought I've configured IDEA to prevent automatically indent my case statement, not sure why it betrays me. Thanks for pointing out!